Harvest.Web/Controllers/PermissionsController.cs

```diff
@@ -0,0 +1,117 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Harvest.Core.Data;
+using Harvest.Core.Domain;
+using Harvest.Web.Models;
+using Harvest.Web.Services;
+using Microsoft.EntityFrameworkCore;
+
+namespace Harvest.Web.Controllers
+{
+    public class PermissionsController : Controller
+    {
+        private readonly AppDbContext _dbContext;
+        private readonly IIdentityService _identityService;
+
+        public PermissionsController(AppDbContext dbContext, IIdentityService identityService)
+        {
+            _dbContext = dbContext;
+            _identityService = identityService;
+        }
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+        public async Task<IActionResult> AddUserRole()
+        {
+            var viewModel = new AddUserRolesModel
+            {
+                Roles = await _dbContext.Roles.Where(a => a.Name != "System").ToListAsync()
+            };
+
+            return View(viewModel);
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> AddUserRole(AddUserRolesModel model)
+        {
+            var viewModel = new AddUserRolesModel
+            {
+                Roles = await _dbContext.Roles.Where(a => a.Name != "System").ToListAsync(),

```

Comment:
Remember to update this once Spruce merges the non magic string version

Reply:
@srkirkland I was reserving System for us. As this page allows Admin access, I don't want system as a choice

Reply:
@jSylvestre Yes but I'm saying the string "System" you should change to RoleCodes.System so it's not just a string